schemas/package.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: package
+  title: Package
+  group: 2
+  short: These fields contain information about an installed software package.
+  description: >
+    These fields contain information about an installed software package. It contains general information about a
+    package, such as name, version or size. It also contains installation details, such as time, URL or
+    user who installed the package.
+  type: group
+  fields:
+
+    - name: name
+      level: core
+      type: keyword
+      description: Package name
+      example: go
+
+    - name: version
+      level: core
+      type: keyword
+      description: Package version
+      example: 1.12.9
+
+    - name: description
+      level: core
+      type: keyword
+      description: Description of the package.
+      example: Open source programming language to build simple/reliable/efficient software.
+
+    - name: size
+      format: string
+      level: core
+      type: long
+      description: Package size in bytes.
+      example: 62231
+
+    - name: installed
+      level: core
+      type: date
+      description: Time when package was installed.
+
+    - name: type
+      level: core
+      type: keyword
+      description: Information about package type. When installed from a package manager, this would contain the
+        package manager name.
+      example: npm
+
+    - name: url.original
+      level: core
+      type: keyword
+      description: URL from where the package was installed.
+      example: https://docs.docker.com/compose/
```

Comment:
Another thing about the URL, I would give an example of a full path to the package archive in full.

If places like Docker Hub don't provide that, perhaps that's acceptable. But I expect this URL to be complete as often as possible. So let's lead by `example:`. Pun intended ;-)

Reply:
After the thread [here](https://github.com/elastic/ecs/pull/532#discussion_r318958329), this might now refer to the "generic URL" of the package and not the URL where it's installed from (what was `remote_repository` when this PR started). If so, the description should be updated to say something neutral like "URL of this package" / "URL with more information about this package".

On where the package was installed from - I don't know any data source (DPKG, RPM, Homebrew, Chocolatey) that provides this? So I would rather keep that out of ECS, and we can always add it later when we have a concrete example and need to add it.

Reply:
Ok, if the exact package source URL isn't available, I'll defer to you here.

But I agree we could perhaps flesh out the description of the field a little more.